deploy/crds/toolchain.dev.openshift.com_toolchainstatuses.yaml

```diff
@@ -119,7 +119,7 @@ spec:
                     description: The status of the host operator's deployment
                     type: string
                   masterUserRecordCount:
-                    description: Number of MasterUserRecords created within the host cluster
+                    description: 'Number of MasterUserRecords created within the host cluster DEPRECATED: use `ToolchainStatusStatus.Metrics` instead'
```

Comment:
Why it's changed? Have we forgotten to update it when changed the toolchain status api?

Reply:
OK. It was not re-generated after https://github.com/codeready-toolchain/api/pull/238/files
There is a PR to update host too https://github.com/codeready-toolchain/host-operator/pull/443 but it still doesn't update the CRD.